apps/src/templates/levelSummary/FreeResponseAIEvaluation.tsx

```diff
@@ -59,31 +54,14 @@ const FreeResponseAIEvaluation: React.FunctionComponent<
       aiEvaluation: aiResponse.aiEvaluation,
       aiReasoning: aiResponse.aiReasoning,
       evaluationCriteria: aiResponse.evaluationCriteria,
+      levelId: levelData.levelId,
+      unitId: levelData.unitId,
+      id: aiResponse.id,
     };
     setEvaluations(prevEvaluations => [...prevEvaluations, evaluation]);
     setEvaluationCount(prevCount => prevCount + 1);
   };
 
-  const summarizeStudentEvaluations = async (
```

Comment:
We currently are not using this anywhere so I removed it.  The `setAISummary` if something went wrong was causing problems because we wanted to assign it an id because it was supposed to be a `AiResponse` type - I thought about just giving it an `id` of `-1` but that felt like it could lead to some problems down the road....